Modelica/Blocks/Types.mo

```diff
@@ -107,11 +107,11 @@ package Types
       input String tableName "Table name";
       input String fileName "File name";
       input Real table[:, :];
-      input SI.Time startTime;
+      input Real startTimeScaled(unit = "1");
```

Comment:
```suggestion
      input Real startTimeScaled;
```
I would prefer it without any unit.
The reason is that there might be users using the external object directly - without the time-scaling.

Reply:
The unit has now been removed.  Resolving.